python/tk_motionbuilder/menu_generation.py

```diff
@@ -263,8 +261,8 @@ def __strip_unicode(self, val):
         """
         Get rid of unicode
         """
-        if val.__class__ == unicode:
-            val = unicodedata.normalize("NFKD", val).encode("ascii", "ignore")
+        if type(val) is six.text_type:
+            val = six.ensure_str(unicodedata.normalize("NFKD", val), "ascii", "ignore")
```

Comment:
Since the code was using unicodedata.normalize which appears to be valid in Python 3, I've opted to keep the check in place, rather than replace the whole thing with a `six.ensure_str()` call.